app/components/application_component.rb

```diff
@@ -7,4 +7,8 @@ class ApplicationComponent < ViewComponent::Base
   def inspect
     "#<#{self.class.name}:#{object_id}>"
   end
+
+  def allow_skip_or_complete?(workflow_step)
```

Comment:
Can you put a comment explaining what this is and where it is used?

Reply:
good idea, adding comment